cmd/iam.go

```diff
@@ -429,27 +429,62 @@ func (sys *IAMSys) DeletePolicy(policyName string) error {
 	return err
 }
 
-// ListPolicies - lists all canned policies.
-func (sys *IAMSys) ListPolicies() (map[string][]byte, error) {
+// InfoPolicy - expands the canned policy into its JSON structure.
+func (sys *IAMSys) InfoPolicy(policyName string) ([]byte, error) {
```

Comment:
There is already an API to fetch the policy definition - `mc admin policy info play readonly` works. This is not needed.

Reply:
It's needed on purpose currently we send all the policies to client which is not useful.

Reply:
Ah yes, server sends all policy definitions and that is what the mc command users. If we are doing this for efficiency, are we also going to modify the list policies to not send the definition any more? I thought that would be excessive changes to the APIs and left it as is.

Reply:
There are also other issues where we don't know which user or group is using the said policy. `mc admin policy list` doesn't provide this information. It is confusing for LDAP users since we want them to use our policy mechanism but they have no way of knowing what they set previously.

So this PR adds breaking changes, I am trying to see if I can do it differently but for now it's not possible without breaking.

Reply:
It is better to list the policies applied to a user or group rather than listing the groups and users with a given policy. It is more direct that way and likely what the a user/operator wants to see.

As mentioned in #8310 we should just allow `mc admin user info` and `mc admin group info` to work - they help determine the policy applied to a user or group and also the groups that a given user is a member of.

Reply:
> As mentioned in #8310 we should just allow `mc admin user info` and `mc admin group info` to work - they help determine the policy applied to a user or group and also the groups that a given user is a member of.

Actually, they already work @donatello - given user being a member of group is not possible

Reply:
does this mean that `mc admin user info` and `mc admin group info` already works in an LDAP setup?

Reply:
Yes they already do according to the code in cmd/iam.go

Reply:
We can modify the doc (https://github.com/minio/minio/blame/master/docs/sts/ldap.md#L262) to reflect that in this PR or can send another one. 

Basically, the change is the following text 
```
 In addition to this, the server will not support operations on users or groups using `mc admin user` or `mc admin group` commands, except `mc admin user info` and `mc admin group info`. This is because users and groups are defined externally in AD/LDAP.
 ```

Maybe another one is better